frontend/src/components/object/ObjectFileDetails.vue

```diff
@@ -185,6 +199,13 @@ watch( [props, getObjects], async () => {
       </div>
       <Divider layout="vertical" />
       <div class="flex flex-column w-6 gap-3 py-5">
+        <div class="flex flex-row-reverse">
+          <ObjectUploadBasic
```

Comment:
wondering if we should be checking for UPDATE permission before showing this button